cmd/kubeadm/app/apis/kubeadm/validation/validation.go

```diff
@@ -70,9 +73,24 @@ func ValidateMasterConfiguration(c *kubeadm.MasterConfiguration) field.ErrorList
 	allErrs = append(allErrs, ValidateToken(c.Token, field.NewPath("token"))...)
 	allErrs = append(allErrs, ValidateFeatureGates(c.FeatureGates, field.NewPath("feature-gates"))...)
 	allErrs = append(allErrs, ValidateAPIEndpoint(c, field.NewPath("api-endpoint"))...)
+	//allErrs = append(allErrs, ValidateProxy(c, field.NewPath("kube-proxy"))...)
```

Comment:
oh one more thing, why is the validation commented here? :)

Reply:
Thanks for the review! This change breaks UT's, I would like to fix the UT's in a follow up PR, then we can turn on this validate of the proxy.

Reply:
ok, let's do that. We'll enable both kubelet and kube-proxy config API validation at the same time then